pkg/database/realm.go

```diff
@@ -1183,8 +1184,8 @@ type RealmUserStats struct {
 
 // CodesPerUser returns a set of UserStats for a given date range.
 func (r *Realm) CodesPerUser(db *Database, start, stop time.Time) ([]*RealmUserStats, error) {
-	start = start.Truncate(time.Hour * 24)
-	stop = stop.Truncate(time.Hour * 24)
+	start = timeutils.Midnight(start)
```

Comment:
utc?

Reply:
Frankly, I am not sure. I've changed this (and above) to UTC, but I rely on other people who understand what we want our system to do more.